client/src/js/nav/components/Icon.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Overlay } from "react-bootstrap";
+import Notifications from "./Notifications";
+import { getSoftwareUpdates } from "../../updates/actions";
+import { getUnbuilt } from "../../indexes/actions";
+
+import { Icon } from "../../base";
+
+class NotificationIcon extends React.Component {
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            show: false
+        };
+    }
+
+    handleToggle = () => {
+
+        if (this.state.show) {
+            window.removeEventListener("click", this.handleExit, false);
+        } else {
+            window.addEventListener("click", this.handleExit, false);
+        }
+
+        this.setState({
+            show: !this.state.show
+        });
+    }
+
+    handleExit = (e) => {
+        if (this.target.contains(e.target)) {
+            return;
+        }
+
+        this.handleToggle();
+    }
+
+    componentWillMount () {
+        this.props.onGet();
+        this.props.onGetUnbuilt();
+    }
+
+    componentDidMount () {
+        this.interval = window.setInterval(() => {
+            this.props.onGet();
+            this.props.onGetUnbuilt();
+        }, 10000);
```

Comment:
Change interval to every few hours or so. Every ten seconds is a lot of requests to _www.virtool.ca_